src/sass/app/components/market.scss

```diff
@@ -327,7 +343,117 @@
+                            flex-direction: row;
+                            flex: 1 0 auto;
+
+                            .symbol_name {
+                                padding: 5px;
+                                margin: 10px 0 5px 5px;
+                                line-height: 18px;
+                                cursor: pointer;
+                                width: 162px;
                                 font-size: 14px;
+                                font-weight: 400;
+                                transition: all 0.2s;
+                                @media (min-width: 320px) and (max-width: 767px) {
+                                    width: 200px;
+                                    font-size: 14px;
+                                }
+
+                                &:hover {
+                                    background: $COLOR_GRAY;
+                                }
+                                &.active {
+                                    background: $COLOR_BLUE;
+                                    color: $COLOR_WHITE;
+                                    @media (min-width: 320px) and (max-width: 767px) {
+                                        background: $COLOR_BLUE;
+                                        color: $COLOR_WHITE;
+                                        font-weight: normal;
+                                        padding-left: 5px;
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+                .subgroup {
+                    display: flex;
+                    flex-direction: column;
+                    flex: 0 0 auto;
+                    padding-bottom: 40px;
+
+                    &:last-of-type {
+                        padding-bottom: 80px;
+                    }
+                    .label {
+                        font-size: 18px;
+                        padding: 10px 0px;
+                        font-weight: 700;
+                        flex: 1 0 auto;
+                        color: $COLOR_BLUE;
+
+                        &.sticky {
+                            position: absolute;
+                            width: 431px;
+                            top: 60px;
+                            background: $COLOR_WHITE;
+                            @media (max-width: 767px) {
+                                top: 123px;
+                                width: calc(100% - 18px);
+                            }
+                        }
+                        &.put_under {
+                            z-index: -1;
+                        }
+                    }
+                    .subgroup-name {
+                        font-size: 18px;
+                        padding: 10px 0;
+                        font-weight: 400;
+                        flex: 1 0 auto;
+                        color: $COLOR_BLUE;
```

Comment:
could we add a mixin for `.subgroup-name` and `.label`? the css is basically the same except for the font-weight